hikari/internal/aio.py

```diff
@@ -176,8 +176,9 @@ async def all_of(
                     await f
                 except asyncio.CancelledError:
                     pass
+
+        gatherer.cancel()
         try:
-            gatherer.cancel()
-            await gatherer
+            await gatherer  # pragma: no cover - I tried everything to make this work with coverage, but no luck :(
```

Comment:
odd, are you able to hit this on a breakpoint?

If so, theres a 0ytest coverage flag to make it more accurate, albeit slower, maybe try that

Reply:
It does hit, the problem is that it only marks it partially covered.

Reply:
![image](https://user-images.githubusercontent.com/29100934/95295558-96db0a80-0877-11eb-8176-fafc37a7eaa8.png)
This is the problem I don't know how to solve

Reply:
Oh, I see the issue.

This will always fail to complete, thus it is saying there is no coverage for the success case, because there cannot be a success case.

One solution would be to use `with contextlib.suppress(asyncio.CancelledError)`, but that is significantly slower than what we have here, and since we may use this function in performance critical places, we should probably avoid that.